boards/arm/blackpill_f401ce/blackpill_f401ce.dts

```diff
@@ -115,3 +115,29 @@
 	pinctrl-0 = <&adc1_in1_pa1>;
 	status = "okay";
 };
+
+&clk_hse {
+	clock-frequency = <DT_FREQ_M(25)>;
+	status = "okay";
+};
+
+&clk_lse {
+	status = "okay";
+};
+
+&pll {
+	div-m = <25>;
+	mul-n = <336>;
+	div-p = <4>;
+	div-q = <7>;
+	clocks = <&clk_hse &clk_lse>;
```

Comment:
why setting also LSE as clock source for the PLL ? The PLL of the stm32F401ce  has HSI or HSE as possible clock inputs.

Reply:
@FRASTM I was afraid I might have done this wrong - the point is that the blackpill_f401ce has an LSE 32768Hz crystal, how do I enable it's use, or should I just disable it?

Reply:
@coderkalyan For now, ​you'd just need to enable the node.
But from what I can see, LSE what not enabled in existing Kconfig configuration. So I'd propose not to do anything regarding LSE ion the default config, at least for now. This is likely because LSE is not required in most cases (at least most of upstream tests and samples).
LSE can be enabled when required in applications that actually need it.